config/config.go

```diff
@@ -45,9 +46,8 @@ type Config struct {
 	KernelArgs            string `json:"kernel_args"`
 	RootDrive             string `json:"root_drive"`
 	CPUTemplate           string `json:"cpu_template"`
-	LogLevel              string `json:"log_level"`
+	LogLevels             string `json:"log_levels"`
```

Comment:
How about using a list (`["debug", "firecracker_error"]`) or  a map (`{"firecracker":"error"}`)? It is odd to use CSV in JSON.

Reply:
@kzys Yea, I'll use the list. Don't know why I defaulted to CSV :x